assets/js/components/ReportError.test.js

```diff
@@ -432,6 +493,6 @@ describe( 'ReportError', () => {
 		expect(
 			getByRole( 'link', { name: /get help/i } )
 		).toBeInTheDocument();
-		expect( queryByText( /retry didn’t work/i ) ).toBeInTheDocument();
+		expect( queryByText( /retry didn't work/i ) ).toBeInTheDocument();
```

Comment:
I think we prefer keeping the apostrophes to be that special character. So this should be an unnecessary change.

Reply:
My VS Code was complaining about it. I have reverted now.